src/js/hca-rjsf/config/form.js

```diff
@@ -670,31 +672,50 @@ const formConfig = {
             'view:spouseIncome': {
               'ui:title': 'Spouse income',
               'ui:options': {
-                hideIf: (formData) => formData.maritalStatus !== 'married' // Something else too?
+                hideIf: (formData) => !formData.maritalStatus || (formData.maritalStatus.toLowerCase() !== 'married') // Something else too?
```

Comment:
I think we should do this case normalization everywhere for `maritalStatus` or just skip it altogether.